README.md

```diff
@@ -26,6 +26,12 @@ exe/generator generate
 exe/generator generate_schema DRO
 ```
 
+## Testing
+
+The generator is tested via it's output when run against `openapi.yml`, viz., the Cocina model classes. Thus, `generate` should be run after any changes to `openapi.yml`.
```

Comment:
I appreciate the clarification here. One nit: `s/it's/its/`

Reply:
Ooh, that is not a grammar error I make. Thanks for catching.